site/docs/concepts/connectors.md

```diff
@@ -531,19 +546,66 @@ captures:
                 6JfAT/AbxeSQF3iBKK8CQAt5r/LLEM1/8ekGOvBh8MAQpWBW771QzHUN84SiUd/q
                 xR9mfItngPwYJ9d/pTO7u9ZUPHEoat8Ave4waB08DsI=
                 -----END RSA PRIVATE KEY-----
-              # Username to connect to the SSH server.
-              user: ssh_user
+        bindings: []
+```
+</TabItem>
+
+<TabItem value="Materialization connectors" default>
+
+```yaml title="materialize-postgres-ssh-tunnel.flow.yaml"
+materializations:
+  acmeCo/postgres-materialize-ssh:
+    endpoint:
+      connector:
+        image: ghcr.io/estuary/materialize-postgres:dev
+        config:
+          # When using a proxy like SSH tunneling, set to localhost
+          host: localhost
+          # Specify an open port on your local machine to connect to the proxy.
+          port: 15432
+          database: flow
+          user: flow_user
+          password: secret
+          networkProxy:
+            sshForwarding:
+              # Port on the local machine from which you'll connect to the SSH server.
+              # If a port is specified elsewhere in the connector configuration, it must match.
+              localPort: 15432
               # Host or IP address of the final endpoint to which you’ll
               # connect via tunneling from the SSH server
               forwardHost: 127.0.0.1
               # Port of the final endpoint to which you’ll connect via
               # tunneling from the SSH server.
               forwardPort: 5432
-              # Port on the local machine from which you'll connect to the SSH server.
-              # This must match port, above.
-              localPort: 15432
-      bindings: []
+              # Location of the remote SSH server that supports tunneling.
```

Comment:
Could add an additional comments of  `in the form of ssh://hostname[:port]` to avoid users using other forms that are not acceptable.